docs/_interface/mining.md

```diff
@@ -1,191 +1,208 @@
+is known as "mining".
 
-The proof-of-work computation can be performed in multiple ways. Geth includes a CPU
-miner, which does mining within the geth process. We discourage using the CPU miner with
-the Ethereum mainnet. If you want to mine real ether, use GPU mining. Your best option for
-doing that is the [ethminer](https://github.com/ethereum-mining/ethminer) software.
+Much more information about mining, including details about the specific algorithm ("Ethash") used by
+Ethereum nodes is available on 
+[ethereum.org](https://ethereum.org/en/developers/docs/consensus-mechanisms/pow/mining). 
 
-Always ensure your blockchain is fully synchronised with the chain before starting to
-mine, otherwise you will not be mining on the correct chain and your block rewards will
-not be valueable.
 
-## GPU mining
+## Mining and the Merge
 
-The ethash algorithm is memory hard and in order to fit the DAG into memory, it needs
-1-2GB of RAM on each GPU. If you get `Error GPU mining. GPU memory fragmentation?` you
-don't have enough memory.
+[The Merge](https://ethereum.org/en/upgrades/merge) is an upcoming upgrade to Ethereum that
+will swap the existing PoW for a [proof-of-stake (PoS)](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos) consensus mechanism. This marks the end of mining on Ethereum. Instead, nodes can [stake ether](https://ethereum.org/en/staking/solo/#get-started-on-the-staking-launchpad) directly and earn ether rewards by running [validators](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/#validators). The merge is expected to happen in the second half of 2022. Until then, Ethereum will continue to be secured by PoW miners. It is no longer recommended to purchase new hardware to participate in Ethereum mining because the chances of returning a profit before The Merge are low.
 
-### Installing ethminer
 
-To get ethminer, you need to install the ethminer binary package or build it from source.
-See <https://github.com/ethereum-mining/ethminer/#build> for the official ethminer
-build/install instructions. At the time of writing, ethminer only provides a binary for
-Microsoft Windows.
+## CPU vs GPU
 
-### Using ethminer with geth
+Participating in Ethereum's PoW mining requires running an algorithm called 
+["Ethash"](https://ethereum.org/en/developers/docs/consensus-mechanisms/pow/mining-algorithms/ethash). Geth includes
+a CPU miner which runs Ethash within the Geth process. This might be useful for mining on some testnets. However, this is CPU mining is not viable on Ethereum Mainnet because CPU miners are easily out-competed by more efficient GPU miners. GPU mining is the recommended method for mining real ether on Ethereum Mainnet, but it is not part of the standard Geth installation. To mine using GPUs an additional piece of third-paty software is required. The recommended GPU mining software is [Ethminer](https://github.com/ethereum-mining/ethminer).
```

Comment:
```suggestion
a CPU miner which runs Ethash within the Geth process. This might be useful for mining on some testnets. However, this is CPU mining is not viable on Ethereum Mainnet because CPU miners are easily out-competed by more efficient GPU miners. GPU mining is the recommended method for mining real ether on Ethereum Mainnet, but it is not part of the standard Geth installation. To mine using GPUs an additional piece of third-party software is required. The recommended GPU mining software is [Ethminer](https://github.com/ethereum-mining/ethminer).
```